### ISSUE TITLE
Figure out new-style tfm multi-targeting for UWP

### ISSUE THREAD

**Issue by ericsink** [ericsink/SQLitePCL.raw#450]
Yes, something in the provider package needs to change… I’m just not sure what. Maybe multi-target net5.0-windows in addition to uap10.0? I’m not sure how MSIX-packaging, appcontainers, and NuGet all interact to pick the assets. I haven’t seen any official guidance yet, but it was pretty sparse in UWP too.

_Originally posted by @bricelam in https://github.com/dotnet/efcore/issues/24213#issuecomment-924454409_

**Comment by ericsink** [comment]
One related thing I have noticed:  I'm using the `uap10.0` TFM all over the place with the assumption that it supports netstandard2.0 when it does not.

For example, when building a UWP-specific provider, I generate a provider with an extra call to a SQLite Windows-specific function, but I don't actually build that provider for the uap10.0 TFM.  Rather, I build it with netstandard2.0.  If I did build it with uap10.0, the reference to SQLitePCLRaw.core would fail, since it builds for netstandard2.0.

When I made the big change to support netstandard2.0 as a minimum, I should have switched the UWP builds to use something like `uap10.0.16299`.

Nowadays, using a TFM like that doesn't seem to work.  The more modern equivalent is `net5.0-windows10.0.16299`, but the net5.0-windows TFMs don't go all the way back to Fall Creators Update, so the earliest one I could support is `net5.0-windows10.0.17763`.  I have no idea what breakage UWP users might see if I changed to that.
